eth/examples/WrappedToken/transfer.ts

```diff
@@ -0,0 +1,16 @@
+import { getContractsFactory } from '../../../src/index';
+/** This example should provide just a method */
```

Comment:
What does this comment mean in the examples?

Reply:
Will remove it.